<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formArticoli</class>
 <widget class="QDialog" name="formArticoli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>591</width>
    <height>320</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Codice Articolo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtArticolo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>9</number>
   </property>
   <property name="placeholderText">
    <string>Codice Articolo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Prezzo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Giacenza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModifica">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modifica</string>
   </property>
   <property name="icon">
    <iconset resource="regcashresource.qrc">
     <normaloff>:/immagini/giacenza.png</normaloff>:/immagini/giacenza.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalva">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Salva</string>
   </property>
   <property name="icon">
    <iconset resource="regcashresource.qrc">
     <normaloff>:/immagini/check.png</normaloff>:/immagini/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnnulla">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
   <property name="icon">
    <iconset resource="regcashresource.qrc">
     <normaloff>:/immagini/close.png</normaloff>:/immagini/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrezzo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGiacenza">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtDescrizione">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Descrizione</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="regcashresource.qrc"/>
 </resources>
 <connections/>
</ui>
